channels/pjsip/dialplan_functions.xml

```diff
@@ -0,0 +1,591 @@
+			the media session for the channel. This will be done using either a
+			re-INVITE (default) or an UPDATE request.
+			</para>
+			<para>This is most useful when combined with the <replaceable>PJSIP_MEDIA_OFFER</replaceable>
+			dialplan function, as it allows the formats in use on a channel to be
+			re-negotiated after call setup.</para>
+			<warning>
+				<para>The formats the endpoint supports are <emphasis>not</emphasis>
+				checked or enforced by this function. Using this function to offer
+				formats not supported by the endpoint <emphasis>may</emphasis> result
+				in a loss of media.</para>
+			</warning>
+			<example title="Re-negotiate format to g722">
+			 ; Within some existing extension on an answered channel
+			 same => n,Set(PJSIP_MEDIA_OFFER(audio)=!all,g722)
+			 same => n,Set(PJSIP_SEND_SESSION_REFRESH()=invite)
+			</example>
+		</description>
+		<see-also>
+			<ref type="function">PJSIP_MEDIA_OFFER</ref>
+		</see-also>
+	</function>
+	<function name="PJSIP_PARSE_URI" language="en_US">
+		<since>
+			<version>13.24.0</version>
+			<version>16.1.0</version>
+			<version>17.0.0</version>
+		</since>
+		<synopsis>
+			Parse an uri and return a type part of the URI.
+		</synopsis>
+		<syntax>
+			<parameter name="uri" required="true">
+				<para>URI to parse</para>
+			</parameter>
+			<parameter name="type" required="true">
+				<para>The <literal>type</literal> parameter specifies which URI part to read</para>
+				<enumlist>
+					<enum name="display">
+						<para>Display name.</para>
+					</enum>
+					<enum name="scheme">
+						<para>URI scheme.</para>
+					</enum>
+					<enum name="user">
+						<para>User part.</para>
+					</enum>
+					<enum name="passwd">
+						<para>Password part.</para>
+					</enum>
+					<enum name="host">
+						<para>Host part.</para>
+					</enum>
+					<enum name="port">
+						<para>Port number, or zero.</para>
+					</enum>
+					<enum name="user_param">
+						<para>User parameter.</para>
+					</enum>
+					<enum name="method_param">
+						<para>Method parameter.</para>
+					</enum>
+					<enum name="transport_param">
+						<para>Transport parameter.</para>
+					</enum>
+					<enum name="ttl_param">
+						<para>TTL param, or -1.</para>
+					</enum>
+					<enum name="lr_param">
+						<para>Loose routing param, or zero.</para>
+					</enum>
+					<enum name="maddr_param">
+						<para>Maddr param.</para>
+					</enum>
+				</enumlist>
+			</parameter>
+		</syntax>
+		<description>
+			<para>Parse an URI and return a specified part of the URI.</para>
+		</description>
+	</function>
+	<function name="PJSIP_HANGUP" language="en_US">
+		<synopsis>
+			Hangup an incoming PJSIP channel with a SIP response code
+		</synopsis>
+		<syntax>
+			<parameter name="response_code" required="true">
+				<para>SIP response code between 400 and 699.</para>
+			</parameter>
+		</syntax>
+		<description>
+			<para>
+			Hangs up an incoming PJSIP channel and returns the
```

Comment:
It should be documented what the behavior of the ISDN cause code is with this

Reply:
Done